loopy/symbolic.py

```diff
@@ -1587,6 +1587,75 @@ def constraint_to_cond_expr(cns):
 # }}}
 
 
+# {{{ isl_set_from_expr
+
+class AffineConditionToISLSet(IdentityMapper):
```

Comment:
Document what type this returns. I don't quite understand what it's supposed to return... Sets, but only sometimes?

Reply:
Done. (Further simplified by only returning isl.Set)